Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/projectblog/ProjectBlog.java

```diff
@@ -0,0 +1,180 @@
+package edu.asu.spring.quadriga.domain.impl.projectblog;
+
+import java.util.Date;
+
+import edu.asu.spring.quadriga.domain.IUser;
+import edu.asu.spring.quadriga.domain.projectblog.IProjectBlog;
+
+/**
+ * This class is concrete implementation of {@linkplain IProjectBlog} interface
+ * and holds the project blog related details.
+ * 
+ * @author Pawan Mahalle
+ *
+ */
+public class ProjectBlog implements IProjectBlog {
```

Comment:
This is for one project blog entry, right? Not a whole blog? Then your class should be named accordingly.